DESCRIPTION

```diff
@@ -1,16 +1,15 @@
 Package: tfpscanner
 Title: Transmission fitness polymorphism scanner 
-Version: 0.1.1
+Version: 0.1.3
```

Comment:
mrc-ide is currently at v0.1.2. There is a conflict between the JR and the MRC-IDE v0.1.1 (two separate v0.1.1s have been created in parallel)

Reply:
Looks good to me :)